skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -312,13 +310,27 @@ actual open class SkiaLayer internal constructor(
     override fun paint(g: java.awt.Graphics) {
         Logger.debug { "Paint called on: $this" }
         checkContentScale()
+        tryRedrawImmediately()
+    }
+
+    override fun setBounds(x: Int, y: Int, width: Int, height: Int) {
```

Comment:
org.jetbrains.skiko.SkiaLayerTest[awt] > resize window[awt] FAILED

fails on Windows

Reply:
Yep, I saw this. Regardless of CI state, please check the fix locally to make sure that we agree that the change makes sense

Reply:
The fix is okay, though I see "jumps" on resize. But it is better than the current behaviour.

https://github.com/JetBrains/skiko/assets/5963351/196783f3-b1d0-484e-8c92-21c81bfd5f40

Regarding the failed test - it fails locally as well, and it is the only failed test.

Reply:
The problem was with software renderer - it just stops to display updates after try to redraw during layout despite any render/flush/... calls after that. Spend some time on it, but the reason isn't clear.

Added a check to apply my fix only to Direct3D to not just fix this issue but also reduce the scope of this change

Reply:
Help with what? The issue was only in Windows/DirectX